test/models_comparison.jl

```diff
@@ -77,21 +67,19 @@ end
 
     v ~ filldist(StickBreakingProcess(crm), K - 1)
     w = stickbreak(v)
-    
-    # Cluster assignments
-    z = tzeros(Int, N)
-    for n = 1:N
-        z[n] ~ Categorical(w)
-    end
 
     # Cluster centers
-    L = identity(K)
+    L = identity(K) # Is this needed?
```

Comment:
Yes, because I want it to produce the same trace structure when I try `maximum(z)` instead.

Reply:
OK, well, it's not actually needed in the PG test.  But it makes switching to `maximum` more explicit.